mindee/parsing/v2/inference_active_options.py

```diff
@@ -5,15 +5,22 @@ class InferenceActiveOptions:
     """Active options for the inference."""
 
     raw_text: bool
+    """Whether raw text extraction is active or not."""
     polygon: bool
+    """Whether polygon extraction is active or not."""
     confidence: bool
+    """Whether confidence scores are active or not."""
     rag: bool
+    """Whether RAG is active or not."""
+    text_context: bool
+    """Whether text context is active or not."""
```

Comment:
use the same wording as in the openapi spec
https://api-v2.mindee.net/docs

<img width="1001" height="560" alt="Image" src="https://github.com/user-attachments/assets/98fb2ea6-3505-45a9-a9d9-5f58937939ec" />